### PR TITLE
Let user specify where to download rollbar js file

### PR DESCRIPTION
**Pull request by adibsaad**

### CODE REVIEW

**Comment by adibsaad** [comment]
@netguru-hound  Did you get a chance to look at this?

**Comment by jniechcial** [comment]
I am on holiday, will take a look beginning of next week

File: index.js [hunk 1]
```diff
@@ -70,11 +70,14 @@ module.exports = {
             }
           };
 
+          var rollbarFileURI = this.readConfig('rollbarFileURI') || 'https://d37gvrvc0wt4s1.cloudfront.net/js/v1.8/rollbar.min.js';
```
Comment (jniechcial) [inline index.js]: Where do you get this default URI from?

File: index.js [hunk 2]
```diff
@@ -49,7 +49,7 @@ module.exports = {
         },
         integrateRollbar: true
       },
-      requiredConfig: ['accessToken', 'accessServerToken', 'minifiedPrependUrl'],
+      requiredConfig: ['accessToken', 'accessServerToken', 'minifiedPrependUrl', 'rollbarFileURI'],
```
Comment (jniechcial) [inline index.js]: You put it in `requiredConfig` while you provide default in `willDeploy`. It should be in `defaultConfig` hash above instead and not here.

**Comment by jniechcial** [comment]
Feature looks interesting, I understand it may be useful. However, I think that it would be better to have the addon using the snippet by default and if necessary download it from web. 

Summing up, if user does not specify `rollbarSnippetURI`, the addon will take the content of the snippet already in the code. If user specify it, it will download and use the content of the snippet.

File: index.js (hunk 1 above)
Comment (adibsaad) [inline index.js]: From the original snippet.js link [here](https://github.com/netguru/ember-cli-deploy-rollbar/blob/master/addon/snippet.js).

File: index.js (hunk 2 above)
Comment (adibsaad) [inline index.js]: My bad, will fix.

**Comment by adibsaad** [comment]
I agree, though I originally only wanted to extract the URL of rollbar.js from snippet.js so that if there's an update, i can easily just update the URL in my config. For example, in this addon's latest version, you are downloading rollbar.js v1.8, but 1.9 is already released. It would be easier maintenance for you if you allowed to specify the rollbar file so you don't have to continuously update it yourself.

**Comment by jniechcial** [comment]
Ok, I fully understand now. But I wonder if the only thing that changes in the official Rollbar.js snippet between versions is this URL. @brianr can you help us and confirm that this is safe way of maintaining the snippet? Because if the snippet is being changed, then the added value is none.

**Comment by adibsaad** [comment]
Bump

**Comment by jniechcial** [comment]
@adibsaad Sorry for late. I've exchange couple of emails with Rollbar and accept the solution - it looks fine. Snippet is rarely changing. Can you resolve conflicts in this branch so I can merge?

**Comment by adibsaad** [comment]
@jniechcial thanks for getting back! I've resolved the conflicts.

**Pull request merged**